### DISCUSSION TITLE
Possible fix of high contrast theme in vscode

### DISCUSSION THREAD

**Discussion by technikhil314** [community/community#6912] [Copilot Conversations] [answered]
I use high contrast theme in vscode but the copilot inline suggestions appear in same color as text, such that I can not differentiate between my code and copilot suggestion. 
Can I know a possible fix for this so I can tweak the theme?

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Hey, you can customize the way we display the inline suggestion by putting the following settings in the settings.json:
```
"workbench.colorCustomizations": {
        "editorGhostText.border": "#ff0000",
        "editorGhostText.foreground": "#ff0000",
    }
```

**Reply by technikhil314** [reply]
Thanks a lot @Krzysztof-Cieslak Let me try this.
